Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -12106,9 +12110,9 @@
 			inputFileListPaths = (
 			);
 			inputPaths = (
-				"$(BUILT_PRODUCTS_DIR)/usr/local/include/WebKitAdditions/Profiling/JavaScriptCore.profdata.compressed",
+				"$(PROFILE_DATA_PATH)",
 				"$(SRCROOT)/../../Tools/Profiling/Empty.profdata",
-				"$(SDK_DIR)/usr/local/include/WebKitAdditions/Profiling/JavaScriptCore.profdata.compressed",
+				"$(SRCROOT)/Scripts/copy-profiling-data.sh",
 			);
 			name = "Copy Profiling Data";
 			outputFileListPaths = (
```

Comment:
Could you please double check that there are output files? Not seeing them on GitHub?

Reply:
Yes, I think GitHub is struggling to expand the common lines of the patch. Here is the whole script phase object:

		F4CDF3C927E9147500191928 /* Copy Profiling Data */ = {
			isa = PBXShellScriptBuildPhase;
			buildActionMask = 12;
			files = (
			);
			inputFileListPaths = (
			);
			inputPaths = (
				"$(PROFILE_DATA_PATH)",
				"$(SRCROOT)/../../Tools/Profiling/Empty.profdata",
				"$(SRCROOT)/Scripts/copy-profiling-data.sh",
			);
			name = "Copy Profiling Data";
			outputFileListPaths = (
			);
			outputPaths = (
				"$(BUILT_PRODUCTS_DIR)/DerivedSources/JavaScriptCore/JavaScriptCore.profdata",
			);
			runOnlyForDeploymentPostprocessing = 0;
			shellPath = /bin/sh;
			shellScript = "if [ \"${ACTION}\" = \"installhdrs\" -o \"${ACTION}\" = \"installapi\" ]; then\n    exit 0;\nfi\n${SCRIPT_INPUT_FILE_2}\n";
		};